transformer/blacklist.py

```diff
@@ -1,25 +1,28 @@
 import os
 import logging
+from typing import Sequence
 
+Blacklist = Sequence[str]
 
-def on_blacklist(url):
-    """
-    Checks for matching URLs in an ignore file (blacklist)
-    from user's current directory.
-    """
+
+def from_file() -> Blacklist:
     blacklist_file = f"{os.getcwd()}/.urlignore"
     try:
         with open(blacklist_file) as file:
-            blacklist = [line.rstrip("\n") for line in file if len(line) > 1]
-
-        for blacklist_item in blacklist:
-            if blacklist_item in url:
-                return True
-
-        return False
-
+            return [line.rstrip("\n") for line in file if len(line) > 1]
     except OSError as err:
         logging.debug(
             "Could not read blacklist file %s. Reason: %s", blacklist_file, err
         )
-        return False
+
```

Comment:
`from_file`'s signature promises to always return a `Blacklist` but this `except` clause makes it return `None`. We should either amend `from_file`'s signature or return an empty `Blacklist`.